service/tbc/tbc.go

```diff
@@ -139,9 +157,13 @@ func headerTime(header []byte) *time.Time {
 	return &h.Timestamp
 }
 
-func hashEqual(h1 chainhash.Hash, h2 chainhash.Hash) bool {
-	// Fuck you chainhash package
-	return h1.IsEqual(&h2)
+func headerHash(header []byte) *chainhash.Hash {
+	h, err := bytes2Header(header)
+	if err != nil {
+		return nil
+	}
+	hash := h.BlockHash()
+	return &hash
 }
 
```

Comment:
The struct `blockPeer` (below this comment, GitHub won't let me comment on it) appears to be unused.
```
service/tbc/tbc.go:174:6: type blockPeer is unused (U1000)
```